components/backstage/components/backstage_category.jsx

```diff
@@ -49,6 +49,7 @@ export default class BackstageCategory extends React.Component {
                 <Route
                     path={link}
                     render={() => (
+                        children && children.length > 0 &&
```

Comment:
Could this go outside of the `Route` component entirely? That way, it wouldn't render anything for the children if there aren't any provided

Reply:
Oh yeah good call, not sure how i missed that :P